src/pages/billing/CreateGCPBillingProject.js

```diff
@@ -30,8 +30,10 @@ const CreateGCPBillingProject = ({
           placeholder: 'Enter a name',
           onChange: v => {
             setBillingProjectName(v)
+            if (!billingProjectNameTouched) {
```

Comment:
I noticed this was emitting an event on every keystroke, which seems quite excessive.

Reply:
Nice find!